crypto/property/property.c

```diff
@@ -288,6 +289,31 @@ static int ossl_method_store_insert(OSSL_METHOD_STORE *store, ALGORITHM *alg)
     return ossl_sa_ALGORITHM_set(store->algs, alg->nid, alg);
 }
 
+/**
+ * @brief Adds a method to the specified method store.
+ *
+ * This function adds a new method to the provided method store, associating it
+ * with a specified id, properties, and provider. The method is stored with
+ * reference count and destruction callbacks.
+ *
+ * @param store Pointer to the OSSL_METHOD_STORE where the method will be added.
+ *        must be non-null.
```

Comment:
Line this up with "Pointer" on the previous line? Also, make the sentence start with a capital.

Reply:
I'll do the alignment, but the sentence starts with the word Pointer, which is capitalized.  store is the name of the parameter which is specified in lowercase.